tests/unit/test_events.py

```diff
@@ -553,23 +553,38 @@ def test_single_run_error():
         event_mgr = TestEventManager()
         ctx_set_event_manager(event_mgr)
 
+        class MockNode:
+            unique_id: str = ""
+            node_info = None
+
         error_result = RunResult(
             status=RunStatus.Error,
             timing=[],
             thread_id="",
             execution_time=0.0,
-            node=None,
+            node=MockNode(),
```

Comment:
`print_run_end_messages()` expects this result to have an actual `node` object, with a `unique_id` value and a `node_info` key